### PR TITLE
Admin panel updates community to ChainType.Chain

### PR DESCRIPTION
**Pull request by kurtisassad**
## Link to Issue
Closes: #5537 

## Description of Changes
- When user adds or updates an RPC endpoint through the admin-panel, it updates the community to ChainType.Chain.

## Test Plan
Front end
1. Create an account
2. In DB upgrade Users table to isAdmin = true
3. Go to localhost:8080/admin-panel
4. check in DB community that is type=off-chain with type ethereum
5. Under the Switch/Add RPC Endpoint, fill out the fields
6. Press update. When successful check type updated to type=chain

Perform the same steps above for cosmos chain, Added a valid unique RPC url. Made sure it loads proposals under the governance section correctly

Back end
- Added case to integration test for backend

### CODE REVIEW

**Comment by mhagel** [comment]
I'm getting an error when I try to upgrade a starter ETH community:
`error
: 
"eth_chain_id is required on ethereum Chain Nodes"
status
: 
400`

Even thought the payload includes `eth_chain_id`

**Comment by kurtisassad** [comment]
Thanks Mark. Should be fixed

File: packages/commonwealth/server/controllers/server_communities_methods/create_chain_node.ts [hunk 1]
```diff
@@ -32,7 +40,7 @@ export async function __createChainNode(
   }
 
   const chainNode = await this.models.ChainNode.findOne({
-    where: { url },
+    where: { [Op.or]: { url, eth_chain_id } },
```
Comment (mhagel) [inline packages/commonwealth/server/controllers/server_communities_methods/create_chain_node.ts:43]: This breaks cosmos upgrades. Maybe this block can be just for EVM?

**Comment by jnaviask** [comment]
cc @kurtisassad where we at with this PR?

**Comment by kurtisassad** [comment]
Drafting for now. I will update it according to marks comment, and resolve conflicts

File: packages/commonwealth/server/controllers/server_communities_methods/create_chain_node.ts (hunk 1 above)
Comment (kurtisassad) [reply packages/commonwealth/server/controllers/server_communities_methods/create_chain_node.ts]: Fixed this issue

**Review by mhagel — CHANGES_REQUESTED** [review]
Left a null-check suggestion. Otherwise looking good.

File: packages/commonwealth/client/scripts/models/ChainInfo.ts
[the remark this answers is not in the document]
Comment (kurtisassad) [reply packages/commonwealth/client/scripts/models/ChainInfo.ts:1]: This was a bug unrelated to this PR. It was fixed on master (which I have merged in here). So it no longer throws that NPE

**Review by mhagel — CHANGES_REQUESTED** [review]
I'm seeing no errors now, but I'm not seeing the successful upgrade to `chain` community anymore.

**Comment by kurtisassad** [comment]
drafting while I retest

**Comment by kurtisassad** [comment]
> I'm seeing no errors now, but I'm not seeing the successful upgrade to chain community anymore.

There seemed to be an issue with the merge conflict but I have fixed it and successfully tested this flow now.

**Review by jnaviask — APPROVED** [review]
Works for me, though we need a better solution for cosmos v1 chains.

**Pull request merged**